backend/src/controllers/serviceControllers.js

```diff
@@ -0,0 +1,46 @@
+import githubService from '../thirdParties/github/githubService';
+
+const getAppropriateService = serviceName => {
```

Comment:
well the if/else makes no sense here hahah
it's returning the same thing eitherwise